src/main/java/org/folio/circulation/resources/DeclareLostResource.java

```diff
@@ -21,15 +21,15 @@
 import io.vertx.ext.web.Router;
 import io.vertx.ext.web.RoutingContext;
 
-public class DeclareLostResource extends Resource {
+public class DeclareLostResource extends AbstractClaimedReturnedResource {
```

Comment:
Hmm... it is a bit confusing now. 

Can we extract note creation logic to some service, e.g. `NoteService` with a `createItemDeclaredLost/ClaimedReturned` methods and then use composition instead of inheritance?

Reply:
I've extracted a note creator class, it isn't quite like the service, but might be similar enough to your thoughts